core/client/fs/src/main/java/alluxio/client/RetryHandlingMetaMasterClient.java

```diff
@@ -45,9 +46,10 @@
    * Creates a new meta master client.
    *
    * @param conf master client configuration
+   * @param alluxioConf Alluxio's configuration
```

Comment:
imo the type is already enough documentation. But "Alluxio's configuration" is worse than nothing because it suggests that there is a single configuration, which is false.

Reply:
I changed them in a later commit